packages/firestore/src/util/assert.ts

```diff
@@ -48,3 +48,11 @@ export function assert(assertion: boolean, message: string): void {
     fail(message);
   }
 }
+
+/**
+ * Return the given value if it is neither null nor undefined, otherwise throws.
+ */
+export function assertExists<T>(value: T | undefined | null): T {
```

Comment:
This should take an optional message parameter? The default message does not provide a lot of value.